### PR TITLE
Tests: Improve tests titles

### PR DESCRIPTION
**Pull request by pro2s**
### Improve tests description ;)
Add a make command to run all tests

cc @vrom911 @chshersh

### CODE REVIEW

**Review by vrom911 — APPROVED** [review]
Thank you for the improvements 👍🏼

File: Makefile [hunk 1]
```diff
@@ -45,3 +45,8 @@ test-chapter4:
 test-chapter4-basic:
 	cabal test doctest-chapter4 --enable-tests --test-show-details=direct
 	cabal run learn4haskell-test --enable-tests -- -m "Chapter4Normal"
+
+.PHONY: test-all
+test-all:
+	cabal test all --enable-tests --test-show-details=direct
+	cabal run learn4haskell-test --enable-tests --
```
Comment (chshersh) [inline Makefile:52]: I think this line is redundant. `cabal test all` should already run all tests, including `doctest`s :slightly_smiling_face:

**Review by chshersh — APPROVED** [review]
@pro2s Thanks! I have only one minor comment, but other than that it looks good :+1:

File: Makefile (hunk 1 above)
Comment (pro2s) [reply Makefile]: Fixed

**Pull request merged**